### DISCUSSION TITLE
オンライン登壇者が落ちたときの対応を検討する

### DISCUSSION THREAD

**Discussion by TakumaNakagame** [cloudnativedaysjp/dreamkast-broadcast#167] [Q&A] [answered]
- [x] 落ちたときに登壇者と連絡をとる手段
- [x] 落ちたときにSY/OBS側で取る対応
- [x] ↑についてなべしまさんとかと合意を取っておく

**Comment by TakumaNakagame** [comment]
>  落ちたときに登壇者と連絡をとる手段
- 基本的にはSlack
- 現状ではコミュニケーション手段がふわっとしているのでちゃんと固める
- PC以外にも、スマホからSlackを使えるようにして欲しい

**Comment by TakumaNakagame** [comment, marked as answer]
> 落ちたときにSY/OBS側で取る対応

### 問題発生時
- 登壇者が落ちたことを検知する → SYのOp/At
  - Opがふた画をSY側で行う
  - Opから近藤さんにトランシーバー経由で連絡
- 登壇者への連絡
  - AtがSlack経由で登壇者へ連絡を取って対応する
- 調整中シーンへの切り替え
  - 近藤さんから指示を受けて、 OBS側で調整中シーンを選択する
- 視聴者への連絡
  - 問題の発生をDkのチャットで連絡したい（現地の人間）
  - 近藤さんから指示を受けて、@TakumaNakagame がチャットに連絡する

### 復帰時
- 全体で調整して復帰するので、勝手に復帰しない
- OBS側の調整シーンを解除する
- StreamYard側のふた画を外す

### アフターフォロー
- Twitterでのお詫び
  - Promotion チャンネルにて、Sharaさんに依頼する

**Comment by nabemasat** [comment]
@TakumaNakagame 昨日は出れずにすみません。確認しました！
